src/domains/wallet/pages/CreateWallet/CreateWallet.test.tsx

```diff
@@ -1,27 +1,57 @@
 /* eslint-disable @typescript-eslint/require-await */
+import { ARK } from "@arkecosystem/platform-sdk-ark";
+import { Environment, Profile, WalletSetting } from "@arkecosystem/platform-sdk-profiles";
 import { act, renderHook } from "@testing-library/react-hooks";
+import { EnvironmentProvider } from "app/contexts";
+import { httpClient } from "app/services";
 import { createMemoryHistory } from "history";
+import nock from "nock";
 import React from "react";
 import { FormContext, useForm } from "react-hook-form";
 import { Route } from "react-router-dom";
 import { fireEvent, render, RenderResult, renderWithRouter, waitFor } from "testing-library";
-import { identity } from "tests/fixtures/identity";
+import { profiles } from "tests/fixtures/env/data";
+import { StubStorage } from "tests/mocks";
 
 import { networks } from "../../data";
 import { CreateWallet, FirstStep, FourthStep, SecondStep, ThirdStep } from "./CreateWallet";
 
-const mnemonic = "lorem ipsum dolor sit amet consectetur";
+let env: Environment;
+let profile: Profile;
 
-const onSubmit = jest.fn();
-const onCopy = jest.fn();
-const onDownload = jest.fn();
+beforeAll(() => {
+	nock.disableNetConnect();
+
+	nock(/.+/)
+		.get("/api/node/configuration/crypto")
+		.reply(200, require("../../../../tests/fixtures/coins/ark/cryptoConfiguration.json"))
+		.get("/api/node/syncing")
+		.reply(200, require("../../../../tests/fixtures/coins/ark/syncing.json"))
+		.get(/\/api\/main_net\/v1\/get_last_transactions_by_address\/.+/)
+		.reply(200, [])
+		.persist();
+});
 
 describe("CreateWallet", () => {
+	beforeEach(async () => {
+		env = new Environment({
+			coins: { ARK },
+			httpClient,
+			storage: new StubStorage(),
+		});
+
+		await env.boot();
```

Comment:
This could be replaced with https://github.com/ArkEcosystem/platform-sdk/commit/031c6008a3f3de3f85349550e6300873a589fbc0 but also works for now as is.

Reply:
Will update, thanks 👌